packages/iov-bns-governance/src/governor.spec.ts

```diff
@@ -0,0 +1,239 @@
+import { Address, Identity, TokenTicker } from "@iov/bcp";
+import { bnsCodec, BnsConnection, VoteOption } from "@iov/bns";
+import { Ed25519HdWallet, HdPaths, UserProfile } from "@iov/keycontrol";
+
+import { Governor } from "./governor";
+import { CommitteeId, ProposalType } from "./proposals";
+
+function pendingWithoutBnsd(): void {
+  if (!process.env.BNSD_ENABLED) {
+    pending("Set BNSD_ENABLED to enable bnsd-based tests");
+  }
+}
+
+// The first IOV key (m/44'/234'/0') generated from this mnemonic produces the address
+// tiov15nuhg3l8ma2mdmcdvgy7hme20v3xy5mkxcezea (bech32) / a4f97447e7df55b6ef0d6209ebef2a7b22625376 (hex).
+// This account has money in the genesis file (see scripts/bnsd/README.md).
+const faucetMnemonic = "degree tackle suggest window test behind mesh extra cover prepare oak script";
+const faucetPath = HdPaths.iov(0);
+const bnsdUrl = "ws://localhost:23456";
+
+async function getConnectionAndIdentity(
+  url = bnsdUrl,
```

Comment:
do we need a configurable `url` argument here?

Reply:
No, this was premature optimisation.